src/server/config/swagger.json

```diff
@@ -1,11 +1,47 @@
 {
   "swagger": "2.0",
   "info": {
-    "version": "1.0",
-    "title": "Final project",
     "description": "API documentation for the final project",
+    "version": "Seasony v1.0",
+    "title": "Final project",
     "contact": {}
   },
-  "host": "",
-  "basePath": "/api"
-}
+  "basePath": "/api",
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "paths": {
+    "/crops": {
+      "get": {
+        "tags": [
+          "crops"
+        ],
+        "summary": "/api/crops",
+        "description": "Get all crops",
+        "operationId": "getAllCrops",
+        "produces": [
```

Comment:
You already said that API can produce application/json, maybe here you can define example response, for example {
  "application/json": {
       "id": "2",
			"name": "Shiso ",
			"plant_variety": "Lamiaceae",
			"fk_user_id": "2",
			"created_at": "2020-03-16 21:03:56",
			"updated_at": "2020-03-16 21:03:56",
			"deleted_at": null
}